packages/yourloops/test/integration/pages/patient-dashboard/patient-dashboard-for-HCP.spec.tsx

```diff
@@ -141,4 +146,178 @@ describe('Patient dashboard for HCP', () => {
+    const dialog = within(screen.getByRole('dialog'))
+    const lowBgInput = dialog.getByRole('spinbutton', { name: 'Low blood glucose input' })
+    const highBgInput = dialog.getByRole('spinbutton', { name: 'High blood glucose input' })
+    const veryLowBgInput = dialog.getByRole('spinbutton', { name: 'Very low blood glucose input' })
+    const outOfRangeTreshold = dialog.getByTestId('basic-dropdown-out-of-range-selector')
+    const hypoTreshold = dialog.getByTestId('basic-dropdown-hypo-threshold-selector')
+    const nonDataThreshold = dialog.getByTestId('basic-dropdown-non-data-selector')
+    const saveButton = dialog.getByTestId('alarm-config-save')
+
+    expect(dialog.getByText('Current trigger setting: 5% of time off target (min at 2.8 mmol/L max at 7.8 mmol/L)')).toBeVisible()
+    expect(dialog.getByText('Current trigger setting: 10% of time below 2.2 mmol/L threshold')).toBeVisible()
+    expect(dialog.getByText('Current trigger setting: 15% of data not transmitted over the period')).toBeVisible()
+    expect(within(dialog.getByTestId('low-bg-text-field-id')).getByText('mmol/L')).toBeVisible()
+    expect(within(dialog.getByTestId('high-bg-text-field-id')).getByText('mmol/L')).toBeVisible()
+    expect(within(dialog.getByTestId('very-low-bg-text-field-id')).getByText('mmol/L')).toBeVisible()
+    expect(lowBgInput).toHaveValue(2.8)
+    expect(highBgInput).toHaveValue(7.8)
+    expect(veryLowBgInput).toHaveValue(2.2)
+    expect(within(outOfRangeTreshold).getByRole('button')).toHaveTextContent('5%')
+    expect(within(hypoTreshold).getByRole('button')).toHaveTextContent('10%')
+    expect(within(nonDataThreshold).getByRole('button')).toHaveTextContent('15%')
+
+    expect(saveButton).not.toBeDisabled()
+
+    await userEvent.clear(lowBgInput)
+    await userEvent.type(lowBgInput, '3.55')
+    expect(within(dialog.getByTestId('low-bg-text-field-id')).getByText('A number with a decimal point')).toBeInTheDocument()
+    expect(saveButton).toBeDisabled()
+    await userEvent.clear(lowBgInput)
+    await userEvent.type(lowBgInput, '4.8')
+    expect(saveButton).toBeEnabled()
+
+    await userEvent.clear(highBgInput)
+    await userEvent.type(highBgInput, '8.55')
+    expect(within(dialog.getByTestId('high-bg-text-field-id')).getByText('A number with a decimal point')).toBeInTheDocument()
+    expect(saveButton).toBeDisabled()
+    await userEvent.clear(highBgInput)
+    await userEvent.type(highBgInput, '8.8')
+    expect(saveButton).toBeEnabled()
+
+    await userEvent.clear(veryLowBgInput)
+    await userEvent.type(veryLowBgInput, '3.55')
+    expect(within(dialog.getByTestId('very-low-bg-text-field-id')).getByText('A number with a decimal point')).toBeInTheDocument()
+    expect(saveButton).toBeDisabled()
+    await userEvent.clear(veryLowBgInput)
+    await userEvent.type(veryLowBgInput, '3.2')
+    expect(saveButton).toBeEnabled()
+    expect(within(outOfRangeTreshold).getByRole('button')).toHaveTextContent('5%')
```

Comment:
`outOfRangeTreshold`, `hypoTreshold` and `nonDataThreshold` have not be changed before saving.